CITATION.md

```diff
@@ -5,3 +5,5 @@ If you use acstools, please cite as:
 #### Other forms of citation
 
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.7406933.svg)](https://doi.org/10.5281/zenodo.7406933)
+
+[![NASA/ADS-ACStools](https://img.shields.io/badge/NASA%2FADS-ACStools-blue)](https://ui.adsabs.harvard.edu/abs/2020ascl.soft11024L/abstract)
```

Comment:
Thanks! Do you want to move this to primary citation section to be consistent with RTD, and move the ASCL one down here?

Reply:
yes, that makes sense. just changed!